geomesa-hbase/geomesa-hbase-tools/bin/aws-utils/.aws-bootstrap-actions.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+#
+# Do not run this script manually. It is part of the automated bootstrap process and is not written for end user use.
+#
+# Core Bootstrap Script for GeoMesa on AWS EMR
+#
+# Note this script runs twice. It is initiated by AWS before the cluster is ready. During its lifetime it downloads a
+# copy of the tools distribution and on the master spawns a clone of itself that waits for the cluster to finish
+# setting up before continuing the bootstrap process.
+
+logFile=/tmp/bootstrap.log
+function log() {
+  timeStamp=$(date +"%T")
+  echo "${timeStamp}| ${@}" | tee -a $logFile
```

Comment:
may be worth adding a geomesa prefix with ``<timestamp> [geomesa]``

Reply:
I'm not convinced that's strictly necessary considering nothing else (even program output) is being written to `/tmp/bootstrap.log`.